### ISSUE TITLE
500 Response from API

### ISSUE THREAD

**Issue by sbw** [searchmysite/searchmysite.net#78]
After #75 closed, API response is again 500.  Example:

https://searchmysite.net/api/v1/search/sbw.org?q=test&page=1&resultsperpage=25

The response headers mention Cloudflare.  Forgive the JSON encoding:

```json
[
 "HTTP\/1.1 500 Internal Server Error",
 "Date: Sat, 15 Oct 2022 14:09:44 GMT",
 "Content-Type: text\/html; charset=utf-8",
 "Connection: close",
 "CF-Cache-Status: DYNAMIC",
 "Report-To: {\"endpoints\":[{\"url\":\"https:\\\/\\\/a.nel.cloudflare.com\\\/report\\\/v3?s=(elided)\"}],\"group\":\"cf-nel\",\"max_age\":604800}",
 "NEL: {\"success_fraction\":0,\"report_to\":\"cf-nel\",\"max_age\":604800}",
 "Server: cloudflare",
 "CF-RAY: 75a9225adf92d01d-SJC",
 "alt-svc: h3=\":443\"; ma=86400, h3-29=\":443\"; ma=86400"
]
```

**Comment by m-i-l** [comment]
I'd tested the q parameter, but not the page and resultsperpage parameters which generated the error (because the strings weren't converted to ints before being used in a calculation). Now resolved, and test cases (#76 ) updated to test the page and resultsperpage parameters.

**Comment by sbw** [comment]
Looks good!  I'm getting valid results from the API now.
